observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,31 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6-1_{context}"]
+== Network Observability Operator 1.6.1
+The following advisory is available for the Network Observability Operator 1.6.1:
+
+* link:https://access.redhat.com/errata/RHSA-2024:4785[2024:4785 Network Observability Operator 1.6.1]
+
+[id="network-observability-operator-CVE-1-6-1_{context}"]
+=== CVEs
+* link:https://access.redhat.com/errata/RHSA-2024:4237[RHSA-2024:4237]
+* link:https://access.redhat.com/errata/RHSA-2024:4212[RHSA-2024:4212]
+
+[id="network-observability-operator-1.6.1-bug-fixes_{context}"]
+=== Bug fixes
+* Previously, packet drops information, such as cause and TCP state, was only available in the Loki datastore and not in Prometheus. For that reason, the drop statistics in the OpenShift web console plugin *Overview* was only available with Loki. Now, packet drops information is also added to metrics, so you can view drops statistics when Loki is disabled. (link:https://issues.redhat.com/browse/NETOBSERV-1649[*NETOBSERV-1649*])
```

Comment:
Similar suggestion about rephrasing in the last sentence:
```suggestion
* Previously, packet drops information, such as cause and TCP state, was only available in the Loki datastore and not in Prometheus. For that reason, the drop statistics in the OpenShift web console plugin *Overview* was only available with Loki. Now, information about packet drops is also added to metrics, so you can view drops statistics when Loki is disabled. (link:https://issues.redhat.com/browse/NETOBSERV-1649[*NETOBSERV-1649*])
```

Although if "packet drops information" is some specific term, then ignore these comments.